frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeTableRow.tsx

```diff
@@ -27,7 +27,7 @@ const CustomServingRuntimeTableRow: React.FC<CustomServingRuntimeTableRowProps>
 }) => {
   const navigate = useNavigate();
   const servingRuntimeName = getServingRuntimeNameFromTemplate(template);
-  const templateOOTB = isTemplateOOTB(template);
+  const templateOOTB = ownedByDSC(template);
```

Comment:
should this be renamed to `ownedByDSC`?

Reply:
I don't feel strongly either way here - the template being owned by the DSC is what considers it out-of-the-box so this is technically correct, but it may be more readable changed to `templateOwnedByDSC`. I think I agree that the function would have been better named `isOwnedByDSC` but it already exists and we'd have to change usages of it in the connection types page - up to you if you want to do that refactor @ppadti